var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -19,29 +19,17 @@ class Coreneuron(CMakePackage):
     git      = "https://github.com/BlueBrain/CoreNeuron"
 
     version('develop', branch='master')
-    version('1.0.2a', commit='c938e4f', submodules=True, preferred=True)
-    version('1.0.1', commit='251ee12', submodules=True)
-    version('1.0.0', tag='1.0', submodules=True)
-    version('1.0b', tag="1.0b", submodules=True)
-    version('1.0a', commit="857551a", submodules=True)
-    version('0.23b', commit="be131ec", submodules=True)
+    # 1.0.1 > 1.0.0.20210519 > 1.0 as far as Spack is concerned
+    version('1.0.0.20210519', commit='c938e4f')
+    version('1.0', tag='1.0')
     version('0.22', tag='0.22', submodules=True)
-    version('0.21a', commit="bf3c823", submodules=True)
-    version('0.20', tag='0.20', submodules=True)
-    version('0.19', tag='0.19', submodules=True)
-    version('0.18', tag='0.18', submodules=True)
-    version('0.17', tag='0.17', submodules=True)
-    version('0.16', tag='0.16', submodules=True)
-    version('0.15', tag='0.15', submodules=True)
-    version('0.14', tag='0.14', submodules=True)
-    patch('0001-Fixes-for-NMODL-MOD2C-binary.patch', when='@0.17+nmodl')
```

Comment:
I have deleted the file for the patch, just in case you think this might be needed somehow.